opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -70,6 +70,47 @@
 ParentSpan = typing.Optional[typing.Union["Span", "SpanContext"]]
 
 
+class Link:
+    """A link to a `Span`."""
+
+    def __init__(
+        self, context: "SpanContext", attributes: types.Attributes = {}
```

Comment:
```suggestion
        self, context: "SpanContext", attributes: types.Attributes = None
```

Reply:
I used that because mypy was complaining, but then lint complained about this one. I had to change types.Attributes to `typing.Optional[typing.Dict[str, AttributeValue]]` and it worked.

Reply:
As to why mypy/lint complain: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments